docs/tutorials/reuse-ssh-connections.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Reuse SSH Connections
+group: advanced-networking
+author: [ "@Dave-Baum" ]
+---
+
+* Table of Contents
+{:toc}
+
+## Intro
+
+Establishing an ssh connection requires a a lot of communication between your host computer
+and the EV3 and can take several seconds.  Although this is fine a long-lived connection,
+it can be quite tedious if you want to use ssh to invoke a single command or scp to transfer
+files.  By instructing ssh to reuse connections, each connection after the first one will
+reuse the session that was already established, drastically speeding up ssh and scp commands.
+
+## Congifuration (OSX, Linux)
```

Comment:
"macOS" is the new "OSX". We use both on the site, so it doesn't matter too much, but I have been trying to standardize on "macOS" since that is now the official name.